src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketCloudTeam.java

```diff
@@ -65,12 +66,12 @@ public void setDisplayName(String displayName) {
 
     @Override
     @JsonIgnore
-    public Map<String, BitbucketHref> getLinks() {
+    public Map<String, List<BitbucketHref>> getLinks() {
         return links;
     }
 
     @JsonIgnore
-    public void setLinks(Map<String, BitbucketHref> links) {
+    public void setLinks(Map<String, List<BitbucketHref>> links) {
```

Comment:
I have changed the method signature in the interface. Not used outside of this plugin so safe to do